styles/typography/typography-preset-7.json

```diff
@@ -0,0 +1,38 @@
+{
+	"version": 3,
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"title": "Literata & Fira Sans",
+	"slug": "typography-preset-7",
+	"styles": {
+		"typography": {
+			"fontFamily": "var:preset|font-family|fira-sans",
```

Comment:
Navigation I see Fira sans at `24px` in the designs, `18px` locally.

For the body size, I see it in the designs at `24px`, and `22px` locally.